app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -146,4 +146,5 @@
 
     <string name="title_summary_record" translatable="false">%1$s (%2$d)</string>
     <string name="start_earlier_end">A data final não pode ser anterior a data inicial</string>
+    <string name="notes">anotações</string>
```

Comment:
Nit: The capitalization is also needed here.

Reply:
Fixed with [0669331](https://github.com/xorum-io/open_money_tracker/pull/34/commits/0669331f398ff184dd02ee2ef21904b2ddad6d2f)